src/lib/replication/repl_dev/raft_repl_dev.cpp

```diff
@@ -470,7 +471,7 @@ void RaftReplDev::check_and_fetch_remote_data(std::vector< repl_req_ptr_t > rreq
 void RaftReplDev::fetch_data_from_remote(std::vector< repl_req_ptr_t > rreqs) {
     if (rreqs.size() == 0) { return; }
 
-    std::vector< ::flatbuffers::Offset< RequestEntry > > entries;
+    std::vector<::flatbuffers::Offset< RequestEntry > > entries;
```

Comment:
Do you have format_on_save turned on?, since it changed the formatting.

Reply:
My formatter is removing the spaces, I have to check the clang installation. Changed it for now